backend/telemetry_core/src/feed_message.rs

```diff
@@ -215,9 +218,11 @@ impl FeedMessageWrite for AddedNode<'_> {
     }
 }
 
+
 #[derive(Serialize)]
 pub struct ChainStatsUpdate<'a>(pub &'a ChainStats);
 
+
```

Comment:
Nit: Unnecessary newlines?

```suggestion
#[derive(Serialize)]
pub struct ChainStatsUpdate<'a>(pub &'a ChainStats);

```